topics/proteomics/tutorials/metaquantome-function/tutorial.md

```diff
@@ -0,0 +1,419 @@
+---
+layout: tutorial_hands_on
+
+title: metaQuantome-Function
+zenodo_link: "https://doi.org/10.5281/zenodo.4110725"
+questions:
+ - "How do I look at differentially expressed proteins?"
+ - "How do I get top 5 or 10 function present in the sample?"
+ - "How can I visualize my results?"
+objectives:
+  - "A functional and quantitational analysis of metaproteomic mass spectrometry data."
+time_estimation: "1h"
+key_points:
+  - "With function and Quantitation inputs perform statistical analysis"
+contributors:
+  - subinamehta
+  - timothygriffin
+  - pratikdjagtap
+  - emmaleith
+  - mariecrane
+  - pravs3683
+
+follow_up_training:
+
+    -
+        type: "internal"
+        topic_name: proteomics
+        tutorials:
+            - metaproteomics
+            
+        type: "internal"
+        topic_name: proteomics
+        tutorials:
+            - metaquantome-data-creation
```

Comment:
if the data creation tutorial belongs before this one, you can define it as a prerequisite instead of follow-up
this follows the same structure, just under `requirements` instead of `follow_up_training` in the metadata:
 
```
requirements:
  -
    type: "internal"
    topic_name: proteomics
    tutorials:
      - metaquantome-data-creation
```

Reply:
Fixed that!